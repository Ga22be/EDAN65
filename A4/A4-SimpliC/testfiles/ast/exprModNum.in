int mod() {
    int a;
    a = 7 % 2;
}