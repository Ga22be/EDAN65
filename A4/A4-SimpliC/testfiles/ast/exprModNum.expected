Program
  List
    Function
      IdDecl ID="mod"
      List
      List
        Declaration
          IdDecl ID="a"
        Assignment
          IdUse ID="a"
          Mod
            Numeral NUMERAL="7"
            Numeral NUMERAL="2"
